### ISSUE TITLE
Apparently undefined behavior with multiple windows and --ontop --blurscreen

### ISSUE THREAD

**Issue by BigMacAdmin** [bartreardon/swiftDialog#230]
Here is the scenario I find myself in:
- I have a project that uses `--listitem` and iterates through a list of actions (Lets call this the Parent Script)
- One of the actions that the Parent Scripts might call is another SwiftDialog script prompting the user for input (Lets call this the Child Script)
- The Parent Script is using `--blurscreen`
- The Child Script is using `--ontop`

The problem, is that the dialog window called by the Child Script can become hidden behind the dialog window called by the Parent Script if the user clicks anywhere outside of the Child Script dialog window. This is a failure condition, because the Parent Script is forever stuck with no way for the user to complete the requested input.

If the Child Script combines the `--blurscreen --ontop` options, then the user can still "hide" the Child Script by clicking the wrong dialog window, but clicking the blurred portion of the screen brings the Child Script back to the foreground.

I want to get my head around expectations of Swift Dialog behavior when there are multiple windows drawn simultaneously. Is this something that has any logic/rules to it, or is it undefined behavior and "admin beware"?

Is it possible for you to change the `--ontop` feature so that this flag work to keep my "Child Script" always on top of the Parent Script?

(As a side note: I've decided that for now I can at least use the `--position` option to ensure my windows never fully overlap/hide each other.)

**Comment by bartreardon** [comment]
yeah, you're in "here be dragons" territory. 

`--ontop` on one window will force it to the top of the stack and instruct the OS to keep it there. Adding a second window with `--ontop` may result in a condition where the OS has to pick which one is the topmost and that behaviour may not be predictable. 

`--blurscreen` is actually a special full screen window that is attached to the dialog that was called with it - anything else will be forced behind the blurred window including any other dialogs that may or may not have `--ontop` defined.

basically - if you're using `--blurscreen` then _that_ dialog process should be the one that receives content update actions.

**Comment by BigMacAdmin** [comment]
Thanks for the info, Bart. I've come up with a strategy to avoid using multiple windows.
